src/main/java/it/gov/pagopa/debtposition/model/enumeration/DebtPositionStatus.java

```diff
@@ -37,4 +41,18 @@ public static Set<DebtPositionStatus> getPaymentPosFullyPaidStatus() {
     public static Set<DebtPositionStatus> getPaymentPosNotAccountableStatus() {
         return EnumSet.of(DRAFT, PUBLISHED, VALID, INVALID, EXPIRED, REPORTED);
     }
+
+    public static PaymentPosition updatePaymentPositionStatus(PaymentPosition pp) {
```

Comment:
Can be void?

Reply:
I prefer that the modifier method returns the modified object, to make the code more readable and fluent ([ref](https://www.baeldung.com/java-fluent-interface-vs-builder-pattern#fluent-interface)). Also, I think that in theory, by keeping this interface, you could also return a copy of the object if necessary.